### ISSUE TITLE
Notification on duplicated builds

### ISSUE THREAD

**Issue by humitos** [readthedocs/readthedocs.org#7177]
We deployed #7123 recently.

Now, I started receiving email notifications about the build being duplicated.

![Screenshot_2020-06-10_17-39-09](https://user-images.githubusercontent.com/244656/84288304-4c53b680-ab41-11ea-8a84-1bc080d821fa.png)

Build: https://readthedocs.org/projects/test-builds/builds/11228342/

I think we should not send these notifications to users.

**Comment by humitos** [comment]
Weird. I received only one notification and I haven't received anymore. I'm sure there were lot of duplicated builds recently and no notifications in my inbox.

**Comment by humitos** [comment]
Check if this is still present after #8815

**Comment by bmw** [comment]
I recently started seeing this for all of [my projects](https://readthedocs.org/profiles/bmw/). It's pretty annoying because I'd like to be notified about build failures, but I'm getting notifications about duplicated builds on every commit.

I tried recreating my webhooks on GitHub but it didn't seem to help. It's not clear to me what's causing the duplicated build, although that seems to be a known problem https://github.com/readthedocs/readthedocs.org/issues/8015.
